cfgov/cfgov/settings/base.py

```diff
@@ -181,6 +183,14 @@
 
 LANGUAGE_CODE = 'en-us'
 
+LANGUAGES = (
+    ('en', _('English')),
+    ('es', _('Spanish')),
+)
+LOCALE_PATHS = (
```

Comment:
Trivial: Add a blank line between these two declarations to match the existing pattern in the surrounding code.